### PR TITLE
Add security-bundle test

### PR DESCRIPTION
**Pull request by fhielpos**
### Checklist

- [x] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

**Review by AverageMarcus — APPROVED** [review]

**Review by stone-z — CHANGES_REQUESTED** [review]
Code is fine but the changelog is wonky

**Comment by fhielpos** [comment]
Lol, don't know what happened when I fixed the conflicts. Now it should be good.

Just waiting on this:
https://tekton.ci.giantswarm.io/#/namespaces/tekton-pipelines/pipelineruns/pr-cluster-test-suites-303-cluster-test-suitesvgsvj?pipelineTask=run-tests&step=run-tests

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add `security-bundle` test in the common/apps.go file.
```
Comment (stone-z) [inline CHANGELOG.md:12]:
```suggestion
- Wait and test for `security-bundle` apps to be deployed.
```

**Review by stone-z — APPROVED** [review]

**Comment by fhielpos** [comment]
Only CAPV failed but is unrelated to this change. Will merge.

**Pull request merged**